### ISSUE TITLE
Would you like to take over as the main maintainer?

### ISSUE THREAD

**Issue by stathissideris** [pepijnve/ditaa#8]
I think the title says it all, I've found it extremely hard to motivate myself in the past few years to maintain ditaa, but interest in it seems to be very much alive. Would you be interested in taking over the official fork?

**Comment by pepijnve** [comment]
My heart says yes, but my head says no. At the moment, I'm finding I don't really have much spare time left over to spend on my open source projects. Taking on maintenance of another one feels like a bad idea. I wouldn't want people to get there hopes up only to be immediately disappointed again.

I'm happy to merge back my changes if you like though if that helps. There's nothing too dramatic in there: stripped away some old cruft, some code cleanup, a couple of minor improvements.

**Comment by stathissideris** [comment]
Well, thank you for even considering it. I like the direction you were taking ditaa, so I thought to ask. You can raise a PR with you changes if you like, but that's exactly the problem: I don't know if/when I'll find the motivation and energy to do something about it!

**Comment by pepijnve** [comment]
What would your proposal be exactly? Keep the code in your repo, but give me push access?

I think I can help out and so some of the maintenance work; I'm just a bit reluctant to make any big announcements since that's might create unrealistic expectations.

**Comment by stathissideris** [comment]
Yes, we could keep the code where it is and give you push access. I would be more than happy with some help clearing up the backlog of PRs and/or issues, and I'd be available to talk about it if you feel that we need to clarify things etc. We can keep it low-profile if you prefer, the only reason I would announce it would be give you the credit you deserve!

**Comment by pepijnve** [comment]
@stathissideris picking up this discussion again due to https://github.com/asciidoctor/asciidoctor-diagram/issues/366
If you grant me push access on your repo, I'm happy to start upstreaming my changes. And fix the SVG CSS issue that was reported; `@font-face` without a `src` attribute doesn't make much sense.
If you prefer, I'm happy to make merge requests for you to review as well.

**Comment by gsingh93** [comment]
@stathissideris Just checking in on whether you're planning to grant @pepijnve push access or not. Would love to see https://github.com/pepijnve/ditaa/issues/1 get merged upstream.

**Comment by tramsauer** [comment]
@stathissideris This would still be very much appreciated (if @pepijnve is still willing to help out :pray: )!
